gensim/models/keyedvectors.py

```diff
@@ -23,7 +23,7 @@
 The vectors can also be instantiated from an existing file on disk
 in the original Google's word2vec C format as a KeyedVectors instance::
 
-  >>> from gensim.models.keyedvectors import KeyedVectors
+  >>> from gensim.models import KeyedVectors
```

Comment:
So, what exactly `KeyedVectors` represents now and why is it moved outside of this module?

Reply:
@janpom this isn't moved, typically, we import all interesting stuff in `__init__.py` of module -> we can import `from gensim.models import KeyedVectors` instead of `from gensim.models.keyedvectors import KeyedVectors`